src/main/java/com/exacttarget/fuelsdk/ETClient.java

```diff
@@ -263,7 +263,10 @@ public synchronized String requestToken(String refreshToken)
         jsonObject.addProperty("clientId", clientId);
         jsonObject.addProperty("clientSecret", clientSecret);
         jsonObject.addProperty("accessType", "offline");
-        if (refreshToken != null) {
+        
+        if (configuration.isFalse("useRefreshToken")) {
```

Comment:
@gwatt Is there a method to see if a value is set in the configuration class?  I am not a fan of the empty if.